translate/rus.json

```diff
@@ -334,10 +334,10 @@
 "resolution":"разрешение",
 "result":"результат",
 "reverse kills":"двойные убийства",
-"reverses":"обратные",
+"reverses":"партий с пер. цветов",
```

Comment:
This translation is a bit better for understanding, but yet a bit too long. Not sure how to short it at the moment.